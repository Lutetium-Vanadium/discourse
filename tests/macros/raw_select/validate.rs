fn main() {
    let q = discourse::questions![RawSelect { validate: todo!() }];
}
