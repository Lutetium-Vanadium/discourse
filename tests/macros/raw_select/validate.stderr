error: option `validate` does not exist for kind `raw_select`
 --> $DIR/validate.rs:2:47
  |
2 |     let q = requestty::questions![RawSelect { validate: todo!() }];
  |                                               ^^^^^^^^
